cocos/2d/CCLayer.cpp

```diff
@@ -266,10 +266,21 @@ void Layer::onAcceleration(Acceleration* acc, Event* unused_event)
 #endif
 }
 
+/////////////////////////////////
+// greentwip fix for key codes //
```

Comment:
That's our organization. I did share the fork with my team and used that comment to give them proper indications about the changes done to the code.